src/NFTBackedToken.sol

```diff
@@ -18,9 +19,10 @@ contract NFTBackedToken is
 {
     IERC721 public nft;
     uint8 public decimals_;
-    uint88 public amountPerNFT;
+    uint88 public amountPerNFT; // with 18 decimals, the max number here is ~309M. should we make this variable larger?
```

Comment:
increased it to 96 bits and improved the order of vars such that this and `nft` are in the same slot